docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-runtime.adoc

```diff
@@ -56,6 +56,24 @@ The following enhancements are planned; the details will be available at release
 |*Exclude Windows vulnerabilities found in Go packages from UNIX OS*
 |Prisma Cloud now excludes vulnerabilities found in Go packages that are specific to Windows from UNIX based operating systems in the Vulnerability Explorer. Select *Monitor > Vulnerabilities > Vulnerability Explorer* for a comprehensive list of the detected vulnerabilities.
 
+//CWP-57947
+//TODO:Pending blurb approval for Kamesh
+|*Support for Google registry scanning*
+|When onboarding GCP cloud accounts, Prisma Cloud now supports scanning of Google Container Registry (GCR) and Google Artifact Registry (GAR).
+
+//CWP-56790
+//TODO:Pending blurb approval for Kamesh
+|*Support for Go packages extraction on Windows*
+|Added support for Go packages extraction on Windows. Package extraction allows the Prisma Cloud Defenders to extract Go packages in zip, tar.gz, or tar.bz2 formats.
+
+//CWP-56788
+//TODO:Pending blurb approval for Kamesh
+|*New Console Environment Variable for system load management*
+|A new Console environment variable, REFRESH_INTERVAL_SECONDS, has been added to prevent system overload issues when using the TAS applications with Defender.
+Previously, Defender checked the state of TAS applications and containers every 10 seconds through an API call to the BBS server. However, frequent changes or restarts in applications within this duration could lead to system overload as Defender scanned all the apps that were impacted by the changes.
```

Comment:
Should we spell out BBS here?

Suggested rewrite:

However, frequent changes or restarts in applications within this duration, could lead to system overload as Defender scans all the apps impacted by the changes.